build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/RunParallelCmds.cs

```diff
@@ -69,7 +69,7 @@ public override bool Execute ()
 							var errOutput = errorOutput.ToString ();
 
 							if (proc.ExitCode != 0) {
-								LogMessage ($"Non-zero exit code: {proc.ExitCode}  Error output: {errOutput}", MessageImportance.High);
+								LogError ($"Non-zero exit code: {proc.ExitCode}  Error output: {errOutput}");
```

Comment:
Should this *also* include the command which failed?  If we're running multiple commands in parallel, it may not be at all obvious *which* command failed.

Reply:
Yes, I added this. 👍